docs/helm.md

```diff
@@ -61,38 +61,44 @@ Kubernetes: `>=1.20`
 | resources | object | `{}` | Specify limits of resources used by the pod |
 | serviceAccount.controller.annotations | object | `{}` | Annotations to add to the Controller ServiceAccount |
 | serviceAccount.snapshot.annotations | object | `{}` | Annotations to add to the Snapshot ServiceAccount |
+| sidecars.attacherImage.additionalArgs | list | `[]` | Customize leaderElection, you can specify `leaseDuration`, `renewDeadline` and/or `retryPeriod`. Each value must be in an acceptable time.ParseDuration format.(Ref: https://pkg.go.dev/flag#Duration) |
+| sidecars.attacherImage.additionalClusterRoleRules | string | `nil` |  |
 | sidecars.attacherImage.enableHttpEndpoint | bool | `false` | Enable http endpoint to get metrics of the container |
 | sidecars.attacherImage.enableLivenessProbe | bool | `false` | Enable liveness probe for the container |
 | sidecars.attacherImage.httpEndpointPort | string | `"8090"` | Port of the http endpoint |
-| sidecars.attacherImage.leaderElection | object | `{}` | Customize leaderElection, you can specify `leaseDuration`, `renewDeadline` and/or `retryPeriod`. Each value must be in an acceptable time.ParseDuration format.(Ref: https://pkg.go.dev/flag#Duration) |
+| sidecars.attacherImage.leaderElection | object | `{}` |  |
 | sidecars.attacherImage.repository | string | `"registry.k8s.io/sig-storage/csi-attacher"` |  |
-| sidecars.attacherImage.tag | string | `"v3.3.0"` |  |
+| sidecars.attacherImage.tag | string | `"v4.7.0"` |  |
```

Comment:
Are we concerned about these changes ?
- [default-fstype](https://github.com/kubernetes-csi/external-attacher/blob/release-4.7/CHANGELOG/CHANGELOG-3.5.md)
- [max_entries](https://github.com/kubernetes-csi/external-attacher/blob/release-4.7/CHANGELOG/CHANGELOG-4.4.md)

Reply:
This change "[max_entries](https://github.com/kubernetes-csi/external-attacher/blob/release-4.7/CHANGELOG/CHANGELOG-4.4.md)" is not breaking because it introduces a new flag, and it does not modify existing functionality. The default behavior remains unchanged unless you explicitly use the --max-entries flag.

Reply:
for the [default-fstype](https://github.com/kubernetes-csi/external-attacher/blob/release-4.7/CHANGELOG/CHANGELOG-3.5.md),we are explicitly specifying ext4 as the default filesystem type (fsType) to be used for provisioning volumes when the StorageClass does not define an fsType. This ensures that our current behavior matches the prior default behavior of the CSI attacher.